aten/src/ATen/native/native_functions.yaml

```diff
@@ -3096,6 +3096,9 @@
 - func: nonzero(Tensor self) -> Tensor
   variants: method, function
 
+- func: nonzero_tuple(Tensor self) -> Tensor[]
+  variants: method, function
```

Comment:
one strange thing here related to the parsing is that this nonzero has an out variant, but nonzero_tuple doesn't.  That seems fine because tuples are immutable in python anyway, but it would be nice to at least test passing an `out` from python and to see that we get a reasonable error message.